portfolio/src/main/webapp/theoffice/theoffice.css

```diff
@@ -1,3 +1,22 @@
+button {
+  background-color: #ffffff; /* Purple */
+  border: 2px solid #800080;
+  color: #000000;
+  cursor: pointer;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  padding: 16px 32px;
+  text-align: center;
+  text-decoration: none;
+  transition-duration: 0.4s;
+}
+
+button:hover {
+  background-color: #800080;
+  color: #ffffff;
```

Comment:
prefer 3 character hexcodes. Here and below. https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation